editor.planx.uk/src/@planx/components/SetFee/Public.tsx

```diff
@@ -1,18 +1,38 @@
 import type { PublicProps } from "@planx/components/shared/types";
-import { makeData } from "@planx/components/shared/utils";
+import { useStore } from "pages/FlowEditor/lib/store";
 import { useEffect } from "react";
 
-import type { SetFee } from "./model";
+import { type SetFee } from "./model";
+import { handleSetFees } from "./utils";
 
 export type Props = PublicProps<SetFee>;
 
 export default function Component(props: Props) {
+  const {
+    applyCalculatedVAT,
+    applyServiceCharge,
+    serviceChargeAmount,
+    applyPaymentProcessingFee,
+  } = props;
+  const fastTrackFeeAmount = Number(props.fastTrackFeeAmount || 0);
```

Comment:
This default value isn't get in `parseSetFee()` because we want the placeholder text to display in the Editor input right?

It might be simpler / cleaner to set the default value of 0 once in `parseSetFee()` and then find a way of using `""` as a value in the input if the fee is 0.

Reply:
Good suggestion, think this is working smoothly now :heavy_check_mark: